docs/install/Knative-custom-install.md

```diff
@@ -283,6 +283,15 @@ commands below.
    kubectl delete deploy knative-ingressgateway -n istio-system
    ```
 
+   You will also need to delete the following resource:
```

Comment:
What do you think about qualifying this paragraph with "If you have installed Knative Eventing Sources, ..." since it only applies to that project?